src/core/counter/mod.rs

```diff
@@ -303,6 +304,11 @@ impl Counter {
 
         Ok(())
     }
+
+    pub fn raw_size(&self) -> Result<u32> {
+        let sizes = sizage(&self.code)?;
+        Ok(sizes.fs)
+    }
```

Comment:
Isn't this always 0, if we are talking about raw size? I am beginning to thing `self.raw().len() as u32` may work in the other two cases.

Reply:
I noticed that the counter size function is incorrect because it must be split into two cases. 
Regarding taking the length instead of computation, I don't know if it always matches.